gapis/api/vulkan/api/command_buffer_control.api

```diff
@@ -272,9 +280,9 @@ enum SemaphoreUpdate {
   @untrackedMap dense_map!(u32, ref!vkCmdWriteTimestampArgs)         vkCmdWriteTimestamp
   @untrackedMap dense_map!(u32, ref!vkCmdCopyQueryPoolResultsArgs)   vkCmdCopyQueryPoolResults
   @untrackedMap dense_map!(u32, ref!vkCmdPushConstantsArgs)          vkCmdPushConstants
-  @untrackedMap dense_map!(u32, ref!vkCmdBeginRenderPassArgs)        vkCmdBeginRenderPass
-  @untrackedMap dense_map!(u32, ref!vkCmdNextSubpassArgs)            vkCmdNextSubpass
-  @untrackedMap dense_map!(u32, ref!vkCmdEndRenderPassArgs)          vkCmdEndRenderPass
+  @untrackedMap dense_map!(u32, ref!vkCmdBeginRenderPassCommonArgs)  vkCmdBeginRenderPass
```

Comment:
Personally, I'd prefer something other than `Common`, because "common" implies that there are some non-common args as well. I'd just call it `vkCmdBeginRenderPassArgs`, or `vkCmdBeginRenderPassXArgs`.

Reply:
As longs as we put something that indicates, this is not a usual Vulkan command, that's fine for me.

I am changing the common to x